tests/e2e/docs/vmc_setup.md

```diff
@@ -0,0 +1,108 @@
+# E2E Tests on VMC environment
```

Comment:
this should be for GC setup right, please change the file name to indicate the same

Reply:
done